docs/pvcscale.md

```diff
@@ -0,0 +1,221 @@
+scenario where many Pods/PVCs are necessary to start in short period of time.
+
+Running from CLI:
+
+```sh
+$ cp workloads/inventory.example inventory
+$ # Add orchestration host to inventory
+$ # Edit vars in workloads/vars/pvcscale.yml or define Environment vars (See below)
+$ time ansible-playbook -vv -i inventory workloads/pvcscale.yml
+```
+
+## Environment variables
+
+### PUBLIC_KEY
+Default: `~/.ssh/id_rsa.pub`
+
+Public ssh key file for Ansible.
+
+### PRIVATE_KEY
+Default: `~/.ssh/id_rsa`
+
+Private ssh key file for Ansible.
+
+### ORCHESTRATION_USER
+Default: `root`
+
+User for Ansible to log in as. Must authenticate with PUBLIC_KEY/PRIVATE_KEY.
+
+### WORKLOAD_IMAGE
+Default: `quay.io/openshift-scale/scale-ci-workload`
+
+Container image that runs the workload script.
+
+### WORKLOAD_JOB_NODE_SELECTOR
+Default: `true`
+
+Enables/disables the node selector that places the workload job on the `workload` node.
+
+### WORKLOAD_JOB_TAINT
+Default: `true`
+
+Enables/disables the toleration on the workload job to permit the `workload` taint.
+
+### WORKLOAD_JOB_PRIVILEGED
+Default: `true`
+
+Enables/disables running the workload Pod as privileged.
+
+### KUBECONFIG_FILE
+Default: `~/.kube/config`
+
+Location of kubeconfig on orchestration host. PVCScale tests supports only
+default location for `kubeconfig` file.
+
+### PBENCH_SSH_PRIVATE_KEY_FILE
+Default: `~/.ssh/id_rsa`
+
+Location of ssh private key to authenticate to the pbench results server.
+
+### PBENCH_SSH_PUBLIC_KEY_FILE
+Default: `~/.ssh/id_rsa.pub`
+
+Location of the ssh public key to authenticate to the pbench results server.
+
+### ENABLE_PBENCH_AGENTS
+Default: `false`
+
+Enables/disables the collection of pbench data on the pbench agent Pods. These Pods are deployed by the workloads/tooling.yml playbook
+it needs to be executed independently of PVCScale test.
+
+### PBENCH_SERVER
+Default: `There is no public default.`
+
+DNS address of the pbench results server.
+
+### SCALE_CI_RESULTS_TOKEN
+Default: `There is no public default.`
+
+Future use for pbench and prometheus scraper to place results into git repo that holds results data.
+
+### JOB_COMPLETION_POLL_ATTEMPTS
+Default: `360`
+
+Number of retries for Ansible to poll if the workload job has completed. Poll attempts delay 10s between polls with some additional time taken for each polling action depending on the orchestration host setup.
+
+### PVCSCALE_TEST_PREFIX
+Default: `pvcscale`
+
+Test to prefix the pbench results.
+
+### PVCSCALE_CLEANUP
+Default: `true`
+
+Enables/disables cleanup of this workload on completion. Cleanup in
+PVCScale terminology means deletion of `pvcscale0` project - this operation can be
+resource intensive in case there are many Pods/PVCs  in `pvcscale0` project which are created
+during test.
+
+### PVCSCALE_BASENAME
+Default: `pvcscale`
+
+Basename used by cluster loader for the project(s) it creates.
+
+### PVCSCALE_MAXPODS
+Default: `1`
```

Comment:
The actual default is `10` in the vars file.  I am not sure which value is more preferable.

Reply:
I tend to keep some low value just for people to run test and get feeling what is going on with this test, let's keep 10 - which is low value ( 10 Pods/PVC on any cluster is fine )